addons/survey/static/src/js/survey_result.js

```diff
@@ -25,26 +25,14 @@ publicWidget.registry.SurveyResultWidget = publicWidget.Widget.extend({
     * @override
     */
     start: function () {
-        var self = this;
         return this._super.apply(this, arguments).then(function () {
```

Comment:
Sorry I didn't say so before but we're trying to remove all `.then()` in favor of `await`, can you squeeze that in as you're here?